examples/secureTokenAndroidKeyStore/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:24.2.0'
     testCompile 'junit:junit:4.12'
+    compile 'io.realm:secure-userstore:1.2.0'
```

Comment:
Already 1.2.0? 😄 
It is possible to delete releases on Bintray and reupload them. Minor issue though